int32[] position_cmd
time stamp

